src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -222,6 +221,43 @@ trait EntityComponent {
       entityAttributeQuery.batchInsertAttributes(attributeRecsToEntityId.keys.toSeq)
     }
 
+    private def upsertAttributes(entities: Traversable[Entity], entityRecs: Traversable[EntityRecord], referencedEntityRecs: Traversable[EntityRecord]) = {
+      def createEntityAttributeTempTable() = {
+        sql"""create temporary table if not exists ENTITY_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, owner_id bigint(20) unsigned NOT NULL)""".as[Int]
```

Comment:
there was discussion about making this a regular create table and dropping the table instead of deleting everything from it